relaton/serializers/bibxml/authors.py

```diff
@@ -67,8 +67,11 @@ def create_author(contributor: Contributor) -> Element:
         else:
             org_el = E.organization(as_list(org.name)[0])
 
-            if org.abbreviation:
+        if org.abbreviation:
+            if isinstance(org.abbreviation, str):
                 org_el.set('abbrev', org.abbreviation)
+            else:
+                org_el.set('abbrev', org.abbreviation.content)
```

Comment:
Same thing here, I can work on normalising the strings in bibxml-service, or we can fix the broken source data (in this case is for IANA and NIST datasets, see https://raw.githubusercontent.com/ietf-tools/relaton-data-iana/main/data/bgp-extended-communities_arp-nd-extended-community-flags.yaml and https://raw.githubusercontent.com/ietf-tools/relaton-data-nist/main/data/NBS_BH_1.yaml ) . @andrew2net could you fix the organization.abbreviation to be a string instead of an object, please?

Reply:
@stefanomunarini  According to RNC spec, I think source data is supposed to be a localized string (and therefore an object): https://github.com/relaton/relaton-models/blob/0d06e95fb2eabfa30d95d274f4055e7b7cb3c943/grammars/biblio.rnc#L157-L158

I think we need to update typings, and in meantime handle the transformation of new source data to old typings.

Reply:
Changed organization.abbreviation type to be a GenericStringValue (AKA LocalizedString) instead, and updated all the references to this field.